File: client/iih_stub/iih.ci_stub/iih.ci.rcm/hospentdto/d/HospEntDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.rcm.hospentdto.d
{
    /// <summary>
    /// HospEntDTO 的摘要说明。
    /// </summary>
    public class HospEntDTO : BaseDTO {

        public HospEntDTO() {
 
        }

        /// <summary>
        /// 病案号
        /// </summary>
		public string Hospital_code {
            get { return getAttrVal<string>("Hospital_code",null); }
            set { setAttrVal<string>("Hospital_code", value); }
        }

        /// <summary>
        /// 科室病区床位
        /// </summary>
		public string Dept_ward_bed {
            get { return getAttrVal<string>("Dept_ward_bed",null); }
            set { setAttrVal<string>("Dept_ward_bed", value); }
        }

        /// <summary>
        /// 病人信息
        /// </summary>
		public string Pat_info {
            get { return getAttrVal<string>("Pat_info",null); }
            set { setAttrVal<string>("Pat_info", value); }
        }

        /// <summary>
        /// 感染日期
        /// </summary>
		public string Infect_date {
            get { return getAttrVal<string>("Infect_date",null); }
            set { setAttrVal<string>("Infect_date", value); }
        }

        /// <summary>
        /// 院感诊断
        /// </summary>
		public string Di_hosp_name {
            get { return getAttrVal<string>("Di_hosp_name",null); }
            set { setAttrVal<string>("Di_hosp_name", value); }
        }

        /// <summary>
        /// 就诊号
        /// </summary>
		public string Id_ent {
            get { return getAttrVal<string>("Id_ent",null); }
            set { setAttrVal<string>("Id_ent", value); }
        }

        /// <summary>
        /// 就诊类型
        /// </summary>
		public string Id_entp {
            get { return getAttrVal<string>("Id_entp",null); }
            set { setAttrVal<string>("Id_entp", value); }
        }

        /// <summary>
        /// 患者号
        /// </summary>
		public string Id_pat {
            get { return getAttrVal<string>("Id_pat",null); }
            set { setAttrVal<string>("Id_pat", value); }
        }

        /// <summary>
        /// 填报日期
        /// </summary>
		public string Dt_report {
            get { return getAttrVal<string>("Dt_report",null); }
            set { setAttrVal<string>("Dt_report", value); }
        }

        /// <summary>
        /// 状态
        /// </summary>
		public string Stages_name {
            get { return getAttrVal<string>("Stages_name",null); }
            set { setAttrVal<string>("Stages_name", value); }
        }

        /// <summary>
        /// 删除原因
        /// </summary>
		public string Delete_reason {
            get { return getAttrVal<string>("Delete_reason",null); }
            set { setAttrVal<string>("Delete_reason", value); }
        }

        /// <summary>
        /// 删除人
        /// </summary>
		public string Delete_person {
            get { return getAttrVal<string>("Delete_person",null); }
            set { setAttrVal<string>("Delete_person", value); }
        }

        /// <summary>
        /// 删除时间
        /// </summary>
		public string Delete_time {
            get { return getAttrVal<string>("Delete_time",null); }
            set { setAttrVal<string>("Delete_time", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Hospital_code", "Dept_ward_bed", "Pat_info", "Infect_date", "Di_hosp_name", "Id_ent", "Id_entp", "Id_pat", "Dt_report", "Stages_name", "Delete_reason", "Delete_person", "Delete_time"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.rcm.hospentdto.d.HospEntDTO";
        }
    }
}

File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/rcm/hospentdto/d/HospEntDTO.java
package iih.ci.rcm.hospentdto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 院感上报患者信息dto DTO数据 
 * 
 */
public class HospEntDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 病案号
	 * @return String
	 */
	public String getHospital_code() {
		return ((String) getAttrVal("Hospital_code"));
	}
	/**
	 * 病案号
	 * @param Hospital_code
	 */
	public void setHospital_code(String Hospital_code) {
		setAttrVal("Hospital_code", Hospital_code);
	}
	/**
	 * 科室病区床位
	 * @return String
	 */
	public String getDept_ward_bed() {
		return ((String) getAttrVal("Dept_ward_bed"));
	}
	/**
	 * 科室病区床位
	 * @param Dept_ward_bed
	 */
	public void setDept_ward_bed(String Dept_ward_bed) {
		setAttrVal("Dept_ward_bed", Dept_ward_bed);
	}
	/**
	 * 病人信息
	 * @return String
	 */
	public String getPat_info() {
		return ((String) getAttrVal("Pat_info"));
	}
	/**
	 * 病人信息
	 * @param Pat_info
	 */
	public void setPat_info(String Pat_info) {
		setAttrVal("Pat_info", Pat_info);
	}
	/**
	 * 感染日期
	 * @return String
	 */
	public String getInfect_date() {
		return ((String) getAttrVal("Infect_date"));
	}
	/**
	 * 感染日期
	 * @param Infect_date
	 */
	public void setInfect_date(String Infect_date) {
		setAttrVal("Infect_date", Infect_date);
	}
	/**
	 * 院感诊断
	 * @return String
	 */
	public String getDi_hosp_name() {
		return ((String) getAttrVal("Di_hosp_name"));
	}
	/**
	 * 院感诊断
	 * @param Di_hosp_name
	 */
	public void setDi_hosp_name(String Di_hosp_name) {
		setAttrVal("Di_hosp_name", Di_hosp_name);
	}
	/**
	 * 就诊号
	 * @return String
	 */
	public String getId_ent() {
		return ((String) getAttrVal("Id_ent"));
	}
	/**
	 * 就诊号
	 * @param Id_ent
	 */
	public void setId_ent(String Id_ent) {
		setAttrVal("Id_ent", Id_ent);
	}
	/**
	 * 就诊类型
	 * @return String
	 */
	public String getId_entp() {
		return ((String) getAttrVal("Id_entp"));
	}
	/**
	 * 就诊类型
	 * @param Id_entp
	 */
	public void setId_entp(String Id_entp) {
		setAttrVal("Id_entp", Id_entp);
	}
	/**
	 * 患者号
	 * @return String
	 */
	public String getId_pat() {
		return ((String) getAttrVal("Id_pat"));
	}
	/**
	 * 患者号
	 * @param Id_pat
	 */
	public void setId_pat(String Id_pat) {
		setAttrVal("Id_pat", Id_pat);
	}
	/**
	 * 填报日期
	 * @return String
	 */
	public String getDt_report() {
		return ((String) getAttrVal("Dt_report"));
	}
	/**
	 * 填报日期
	 * @param Dt_report
	 */
	public void setDt_report(String Dt_report) {
		setAttrVal("Dt_report", Dt_report);
	}
	/**
	 * 状态
	 * @return String
	 */
	public String getStages_name() {
		return ((String) getAttrVal("Stages_name"));
	}
	/**
	 * 状态
	 * @param Stages_name
	 */
	public void setStages_name(String Stages_name) {
		setAttrVal("Stages_name", Stages_name);
	}
	/**
	 * 删除原因
	 * @return String
	 */
	public String getDelete_reason() {
		return ((String) getAttrVal("Delete_reason"));
	}
	/**
	 * 删除原因
	 * @param Delete_reason
	 */
	public void setDelete_reason(String Delete_reason) {
		setAttrVal("Delete_reason", Delete_reason);
	}
	/**
	 * 删除人
	 * @return String
	 */
	public String getDelete_person() {
		return ((String) getAttrVal("Delete_person"));
	}
	/**
	 * 删除人
	 * @param Delete_person
	 */
	public void setDelete_person(String Delete_person) {
		setAttrVal("Delete_person", Delete_person);
	}
	/**
	 * 删除时间
	 * @return String
	 */
	public String getDelete_time() {
		return ((String) getAttrVal("Delete_time"));
	}
	/**
	 * 删除时间
	 * @param Delete_time
	 */
	public void setDelete_time(String Delete_time) {
		setAttrVal("Delete_time", Delete_time);
	}
}